test/modules/TwinTester/TwinEdgeOperationsInitializer.cs

```diff
@@ -36,8 +36,6 @@ public static async Task<TwinEdgeOperationsInitializer> CreateAsync(RegistryMana
 
                 // reset reported properties
                 await TwinTesterUtil.ResetTwinReportedPropertiesAsync(moduleClient, twin);
-                
-
                 return new TwinEdgeOperationsInitializer(registryManager, moduleClient, reporter, 0);
```

Comment:
Please revert the change to this file...

Reply:
I believe that the best way to do this is to squash your change, do a `git reset HEAD~1` and then checkout this file.